src/main/java/net/minestom/server/entity/MetadataHolder.java

```diff
@@ -132,7 +132,7 @@ public void setNotifyAboutChanges(boolean notifyAboutChanges) {
 
     static final Map<String, BiFunction<Entity, MetadataHolder, EntityMeta>> ENTITY_META_SUPPLIER = createMetaMap();
 
-    static EntityMeta createMeta(EntityType entityType, Entity entity, MetadataHolder metadata) {
+    public static EntityMeta createMeta(EntityType entityType, Entity entity, MetadataHolder metadata) {
```

Comment:
ApiStatus.Internal, NotNull, Nullable, NotNull.

I am fine with this as a temporary solution for doing weird things with metadata, but I do not want to encourage it.